Source/UETorch/Private/UETorch.cpp

```diff
@@ -706,3 +706,43 @@ extern "C" bool CaptureDepthField(UObject* _this, const IntSize* size, void* dat
 	}
 	return true;
 }
+
```

Comment:
We don't need comments for each for each of these functions but please add at least a section comment saying "Getters and setters for Actor properties".